Makefile

```diff
@@ -72,6 +72,25 @@ else
 	export GOPATH := $(shell go env GOPATH)
 endif
 
+#
+# Ginkgo configuration.
+#
+GINKGO_FOCUS ?=
+GINKGO_SKIP ?=
+GINKGO_NODES ?= 1
+GINKGO_TIMEOUT ?= 2h
+GINKGO_POLL_PROGRESS_AFTER ?= 10m
+GINKGO_POLL_PROGRESS_INTERVAL ?= 1m
+E2E_CONF_FILE ?= config/docker.yaml
```

Comment:
Should we make this based on the repo-root?

Essentially use it as ~ `$(ROOT_DIR)/$(E2E_CONF_FILE)` and change the value here to ./test/e2e/config/docker.yaml?

Seems more intuitive for users of this makefile